src/parse.rs

```diff
@@ -38,6 +38,30 @@ use crate::tree::{Tree, TreeIndex, TreePointer};
 // https://spec.commonmark.org/0.29/#link-destination
 const LINK_MAX_NESTED_PARENS: usize = 5;
 
+/// Codeblock kind.
+#[derive(Clone, Debug, PartialEq)]
+pub enum CodeBlockKind<'a> {
```

Comment:
This definition should probably be re-exported [here](https://github.com/raphlinus/pulldown-cmark/blob/master/src/lib.rs#L73), as there is currently no way to extract the language string.